src/js/facility-locator/components/search-results/FacilityDirectionsLink.jsx

```diff
@@ -10,6 +10,7 @@ class FacilityDirectionsLink extends Component {
     return (
       <span>
         <a href={`https://maps.google.com?saddr=Current+Location&daddr=${addressArray.join(', ')}`} target="_blank">
+          <span className="usa-sr-only">External link</span>
```

Comment:
@markgreenburg is this consistent with the planned link accessibility approach?

Reply:
Thanks for investigating content solutions for external links @peggygannon, it looks like we will use the existing external link background image that applies for all external links (as confirmed by @stacyhlinn) and now include a hidden span for screen readers only, according to best practices:
- [Links to New Windows, Pop-ups, Other Frames, or External Web Sites](https://webaim.org/techniques/hypertext/hypertext_links#new_window)
- [Background images](https://webaim.org/techniques/alttext/#background)

Reply:
@cehsu I took a CSS-only approach due to 1.) not wanting to add spans for all the existing links and 2.) feeling that we may fail to consistently apply the correct markup going forward given that we don't currently have any checks / tests in place. Here's essentially what I came up with:
```

Reply:
Also for reference, here's the PR (still waiting on some input but should give you an idea of how I think I could solve for our external links specifically) https://github.com/department-of-veterans-affairs/vets-website/pull/6863

Also, note that I don't love this approach as a long-term solution - I think markup is the best way to go.

Reply:
That makes sense and thanks for sharing this additional approach @markgreenburg.
It seems like this may contradict some guidance, but is a good initial step until we update all external links and links that open in new tabs:
>Sometimes image sprites or other background images are used to present content. While these should generally be avoided, if they are used, any content conveyed through the background image must be made accessible within the page markup. If the PDF icon image in Example 7 above were instead presented via a CSS background image, you could use a text replacement technique to present the content within the link:
<a href="application.pdf">Download the Employment Application<span class="pdficon"> (PDF)</span></a>
You would then use CSS to position the "(PDF)" text (the span class="pdficon" element) off-screen. See CSS in Action: Invisible Content Just for Screen Reader Users for more details. Sighted users would see the PDF background image and screen reader users would hear the off-screen "(PDF)" text.
Source: https://webaim.org/techniques/alttext/
One of the considerations here is that background images may be disabled in high-contrast display modes.

Reply:
@markgreenburg Why do you prefer markup over the css solution?